apps/tsilhqotin-language-hub/src/app/App.tsx

```diff
@@ -1,26 +1,28 @@
 import Typography from '@mui/material/Typography';
-import { HashRouter, Route, Routes } from 'react-router-dom';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import About from '../Components/Pages/About/About';
 import Apps from '../Components/Pages/Apps/Apps';
 import Funders from '../Components/Pages/Funders/Funders';
 import Greetings from '../Components/Pages/Greetings/Greetings';
 import Home from '../Components/Pages/Home/Home';
 import Links from '../Components/Pages/Links/Links';
 import Videos from '../Components/Pages/MediaViewModel/MediaViewModel';
-import Songs from '../Components/Pages/SongViewModel/SongViewModel';
+import SongDetail from '../Components/Pages/SongDetail/SongDetail';
+import Songs from '../Components/Pages/SongIndex/SongIndex';
 import Teachers from '../Components/Pages/Teachers/Teachers';
 import Navbar from '../Components/Widgets/Navbar/Navbar';
 
 export function App() {
     return (
         <Typography component={'span'} variant={'body2'}>
-            <HashRouter>
+            <BrowserRouter>
                 <Navbar />
                 <Routes>
                     <Route path="/" element={<Home />} />
                     <Route path="/About" element={<About />} />
                     <Route path="/Apps" element={<Apps />} />
                     <Route path="/Songs" element={<Songs />} />
+                    <Route path="/Songs/:id" element={<SongDetail />} />
```

Comment:
Good. In the future, we may want to use a pattern to actually compute the index route from the detail route.